layouts/hub.tsx

```diff
@@ -99,7 +83,45 @@ const HubLayout = ({ children }: { children: React.ReactNode }) => {
               </div>
             </>
           ) : (
-            <></>
+            <>
+              {validatePath(pathname) ? (
```

Comment:
What's the `validatePath` for? What's the case for when it's false?

Reply:
It is used to prevent the header from displaying in other pages the uses same layout like `hub/insights/new` and `hub/lists/new`. if the test fails this pattern `/^\/hub\/(insights|lists)?$/`, the header will be hidden.